wire-ios-data-model/Source/Model/Message/ConversationMessage.swift

```diff
@@ -293,6 +296,13 @@ extension ZMMessage: ZMConversationMessage {
 
         return fileSharingFeature.status == .disabled
     }
+
+    public var failedToSendUsers: [UserType]? {
+        guard let recipients = failedToSendRecipients else {
+            return nil
+        }
+        return Array(recipients)
+    }
```

Comment:
Could we make this non optional and return an empty array in case there are no users?

Reply:
Yes, done